### PR TITLE
disable_verify_server_general

### PR DESCRIPTION
**Pull request by cspiel1**

### CODE REVIEW

File: include/re_tls.h [hunk 1]
```diff
@@ -63,6 +63,7 @@ int tls_set_verify_server(struct tls_conn *tc, const char *host);
 
 int tls_get_issuer(struct tls *tls, struct mbuf *mb);
 int tls_get_subject(struct tls *tls, struct mbuf *mb);
+void tls_enable_sverify(struct tls *tls, bool enable);
```
Comment (sreimers) [inline include/re_tls.h:66]: I think `tls_disable_server_verify(struct tls *tls)` and if needed `tls_enable_server_verify(struct tls *tls)` would be more clear.

File: src/tls/openssl/tls.c [hunk 2]
```diff
@@ -142,6 +143,7 @@ int tls_alloc(struct tls **tlsp, enum tls_method method, const char *keyfile,
 	if (!tls)
 		return ENOMEM;
 
+	tls->sverify = true;
```
Comment (sreimers) [inline src/tls/openssl/tls.c:146]: i think `tls->server_verify` is easier to read and remember

File: include/re_tls.h (hunk 1 above)
Comment (sreimers) [reply include/re_tls.h:66]: maybe more consistent would be `tls_disable_verify_server`

File: src/tls/openssl/tls.c (hunk 2 above)
Comment (sreimers) [reply src/tls/openssl/tls.c:146]: more consistent `verify_server`

Comment (cspiel1) [reply src/tls/openssl/tls.c:146]: ok.

**Pull request merged**